src/material-experimental/mdc-chips/_chips-theme.scss

```diff
@@ -1,93 +1,163 @@
-@use '@material/chips/deprecated' as mdc-chips;
+@use '@material/chips/chip' as mdc-chip;
+@use '@material/chips/chip-theme' as mdc-chip-theme;
+@use '@material/chips/chip-set' as mdc-chip-set;
 @use '@material/theme/theme-color' as mdc-theme-color;
+@use '@material/theme/color-palette' as mdc-color-palette;
 @use 'sass:color';
 @use 'sass:map';
 @use '../mdc-helpers/mdc-helpers';
 @use '../../material/core/typography/typography';
 @use '../../material/core/theming/theming';
 
-@mixin _selected-color($color) {
-  @include mdc-chips.fill-color($color, $query: mdc-helpers.$mat-theme-styles-query);
-  @include mdc-chips.ink-color(text-primary-on-dark, $query: mdc-helpers.$mat-theme-styles-query);
-  @include mdc-chips.selected-ink-color-without-ripple_(
-                  text-primary-on-dark,
-                  $query: mdc-helpers.$mat-theme-styles-query
-  );
-  @include mdc-chips.leading-icon-color(text-primary-on-dark,
-    $query: mdc-helpers.$mat-theme-styles-query);
-  @include mdc-chips.trailing-icon-color(text-primary-on-dark,
-    $query: mdc-helpers.$mat-theme-styles-query);
+@mixin _base-chip($is-dark) {
+  $on-surface-state-content: if($is-dark, mdc-color-palette.$grey-50, mdc-color-palette.$grey-900);
+  $surface:
+    color.mix(mdc-theme-color.prop-value(on-surface), mdc-theme-color.prop-value(surface), 12%);
+
+  // TODO(crisbeto): ideally the $label-text-color would be something along the lines of:
+  // `if($is-dark, mdc-color-palette.$grey-100, mdc-color-palette.$grey-800)`, because it allows
+  // the user to distinguish between when the primary action is focused versus the remove icon.
+  // The problem is that the color contrast is worse and there are a lot of g3 screenshot tests
```

Comment:
I would try to write out "Google" rather than "g3" so that it's clear to passerby